.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+*   @aws-amplify/android-team
```

Comment:
This shows me unknown owner, probably because this is in different repo (`awslabs` instead of `aws-amplify`)

Reply:
Yupp already on it.